src/app/pages/Collections/Collections.tsx

```diff
@@ -128,9 +143,33 @@ export default function Collections(): JSX.Element {
               searchValue={addNewCollection}
               onChange={(event) => setAddNewCollection(event.target.value)}
             />
+            <AddButton
+              className={styles.addButtonPos}
+              type="button"
+              onClick={() => setModalToggle(false)}
```

Comment:
Maybe name this state to reflect which modal is meant, you do that with the `deleteModal`